customTests/azure_nccl_allreduce_ib_loopback.nhc

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 # Check for IB issues by running NCCL allreduce disabling NCCL shared memory.
+ALL_REDUCE_EXE_PATH=/opt/nccl-tests/build/all_reduce_perf
```

Comment:
Any reason not to use the same var name for this value in each file?

Reply:
The reason is NHC sources all the custom test scripts before running the associated checks. So if the variables are the same then they will get over written. Although, the paths are the same. I made a decision to name them differently to make sure the tests are isolated from each other.